packages/core/core/src/Graph.js

```diff
@@ -6,18 +6,19 @@ import type {TraversalActions, GraphVisitor} from '@parcel/types';
 import assert from 'assert';
 import nullthrows from 'nullthrows';
 
-export type GraphOpts<TNode, TEdgeType: string | null = null> = {|
+type NullEdgeType = 0;
```

Comment:
I wonder if we should not support the null edge type at all, and just have users/extenders always specify the types of edges the way they do nodes.

Reply:
cc @devongovett @mischnic